Predictor_submission_template.ipynb

```diff
@@ -274,19 +286,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def predict(start_date: str, end_date: str, path_to_ips_file: str):\n",
+    "def predict(start_date: str,\n",
+    "            end_date: str,\n",
+    "            path_to_ips_file: str,\n",
+    "            output_file_path) -> None:\n",
     "    \"\"\"\n",
-    "    Generates a file with daily new cases predictions for the given countries, regions and npis, between\n",
-    "    start_date and end_date, included.\n",
+    "    Generates and saves a file with daily new cases predictions for the given countries, regions and intervention\n",
+    "    plans, between start_date and end_date, included.\n",
     "    :param start_date: day from which to start making predictions, as a string, format YYYY-MM-DDD\n",
     "    :param end_date: day on which to stop making predictions, as a string, format YYYY-MM-DDD\n",
-    "    :param path_to_ips_file: path to a csv file containing the intervention plans between start_date and end_date\n",
-    "    :return: Nothing. Saves a csv file called 'start_date_end_date.csv'\n",
+    "    :param path_to_ips_file: path to a csv file containing the intervention plans between inception date (Jan 1 2020)\n",
+    "     and end_date, for the countries and regions for which a prediction is needed\n",
```

Comment:
`path to file to which to save the the predictions` -> `path to file to save the predictions to`

Reply:
FTR The original version was grammatically correct. It's just a matter of preference and style. I won't say which one I prefer :D

Reply:
Some would say Elliot's version is "wrong" because of an archaic non-rule about not ending sentences with a preposition. But that rule in itself is BS, born of misguided Victorian grammarians desperately trying to make English resemble Latin. There's nothing wrong with ending sentences with a preposition, even though for some strange reason it's considered "informal". And no I'm not going to paste the (apocryphal) Churchill quote about prepositions!

Reply:
yes, except the preposition at the end was a bit of misdirection ;) the main point is to get rid of the the the.

Reply:
Fixed, as well as in `predict.py` and the examples `predict.py`

Reply:
Haha I didn't even notice the doubled "the". Real-world example of the [Paris in the Spring](https://en.wikipedia.org/wiki/Paris_in_the_Spring) test.